### ISSUE TITLE
`order` for destructured?

### ISSUE THREAD

**Issue by pvinis** [import-js/eslint-plugin-import#2364]
in the `order` rule examples, there is
```
import { compose, apply } from 'xcompose';
```

Is it possible to also order the destructured names, so that the above is wrong and the below is correct?
```
import { apply, compose } from 'xcompose';
```

Thanks.

**Comment by ljharb** [comment]
No, there’s no rule to sort the names within an import statement

The core rule https://eslint.org/docs/rules/sort-imports does handle this, though.

**Comment by soryy708** [comment]
Since this is already handled by an ESLint core rule, there's no need to duplicate this in the plugin.
